src/app/features/home/custom-camera/custom-camera.page.ts

```diff
@@ -287,6 +291,50 @@ export class CustomCameraPage implements OnInit, OnDestroy {
     }
   }
 
+  private async convertFilePath(file: PickedFile): Promise<string | undefined> {
+    /**
+     * WORKAROUND: https://github.com/numbersprotocol/capture-lite/issues/2857
+     * Convert Android picked image file path to "file:///" format for editing and saving.
+     *
+     * Here is the explanation of the WORKAROUND:
+     * We have a CapacitorJS project where we encounter an issue with picking images
+     * from the gallery on Android. When users pick media from the gallery on iOS, the
+     * file path is in the format "file:///", while on Android it is in the format
+     * "content:///". This discrepancy in file paths poses a challenge for us as we
+     * need to perform additional pre-publish editing, such as cropping and applying a
+     * grayscale filter, on the picked images. However, to save the edited image to the
+     * file system, we require the file path to be in the "file:///" format instead of
+     * "content:///".
+     *
+     * To address this issue, we have implemented a workaround that involves copying
+     * the picked image to the file system, ensuring that the file path is in the
+     * required "file:///" format. This workaround allows us to seamlessly perform the
+     * necessary editing operations on the image and successfully save it to the file
+     * system.
+     *
+     * Note: This workaround is specifically designed to handle picked images from the
+     * gallery on Android and does not impact the functionality on iOS.
+     *
+     * Please make sure to adjust the placeholders ("<issue link>",
+     * "<workaround-purpose>") with the relevant information in your specific context.
```

Comment:
Is this section supposed to be removed?

Reply:
@shc261392, yep will remove and re-commit.

Reply:
done [chore(custom-camera.page): remove unused comment](https://github.com/numbersprotocol/capture-lite/pull/2869/commits/28aa6709177061f0308c58e7fc14da9c73c25dd8)